be/src/olap/in_list_predicate.h

```diff
@@ -76,9 +76,11 @@ struct equal_to<doris::uint24_t> {
 
 namespace doris {
 
-template <class T, PredicateType PT>
+template <PrimitiveType Type, PredicateType PT>
 class InListPredicateBase : public ColumnPredicate {
 public:
+    using T = std::conditional_t<Type == TYPE_DATE, uint24_t,
```

Comment:
We should unify  CppType ---> PrimitiveType relation. Pls do not define them everywhere.